Formula/apache-flink.rb

```diff
@@ -0,0 +1,21 @@
+class ApacheFlink < Formula
+  desc "Scalable batch and stream data processing"
+  homepage "https://flink.apache.org/"
+  url "https://www.apache.org/dyn/closer.lua?path=flink/flink-1.0.2/flink-1.0.2-bin-hadoop26-scala_2.10.tgz"
+  version "1.0.2"
+  sha256 "a3d68b3d8d90c742f5a137daec1ea38c28ed6e8498f6e1b5430a54b4621ca989"
+  head "https://github.com/apache/flink.git"
+
+  bottle :unneeded
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install Dir["*"]
+    bin.write_exec_script Dir["#{libexec}/bin/{flink,pyflink2.sh,pyflink3.sh}"]
+  end
+
+  test do
+    ENV["_JAVA_OPTIONS"] = "-Djava.io.tmpdir=#{testpath} -Duser.home=#{testpath}"
+    assert_match /FINISHED/, shell_output("#{libexec}/bin/start-scala-shell.sh local <<<'env.fromElements(1,2,3).print()'", 1)
```

Comment:
@mikemcquaid I don't see any residual processes after the test run - the Flink shell doesn't leave any as far as I know.   Can you elaborate on your concern?

I'd like to use one the `bin` scripts, yes, but I didn't see how to add `start-scala-shell` to `bin` without also adding it to the system path.  I don't want to add `start-scala-shell` to the system path at this time.  Are the actions separable?

Thanks